Doc/library/urllib.parse.rst

```diff
@@ -37,16 +37,22 @@ URL Parsing
 The URL parsing functions focus on splitting a URL string into its components,
 or on combining URL components into a URL string.
 
-.. function:: urlparse(urlstring, scheme='', allow_fragments=True)
+.. function:: urlparse(url, scheme='', allow_fragments=True)
 
-   Parse a URL into six components, returning a 6-item :term:`named tuple`.  This
-   corresponds to the general structure of a URL:
+   Parse a URL into six components, returning a 6-item :term:`named tuple`.
+   This corresponds to the general structure of a URL:
    ``scheme://netloc/path;parameters?query#fragment``.
-   Each tuple item is a string, possibly empty. The components are not broken up
-   into smaller parts (for example, the network location is a single string), and %
-   escapes are not expanded. The delimiters as shown above are not part of the
-   result, except for a leading slash in the *path* component, which is retained if
-   present.  For example:
+
+   The delimiters as shown above are not part of the result, except for a
+   leading slash in the path component, which is retained if present.
+
+   Additionally, the netloc property is broken down into these additional
+   attributes added to the returned object: username, password, hostname, and
+   port.
```

Comment:
The same information is given in the below table, so I'm not sure how necessary this paragraph is?

I don't see harm in having it, though.

The field names should be formatted, though:

```
   Additionally, the netloc property is broken down into these additional
   attributes added to the returned object: :attr:`username`, :attr:`password`, :attr:`hostname`,
   and :attr:`port`.
```